conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -490,35 +490,39 @@ TEST_F(
         }
       }
 
-      if (power_sustained_set.limitValueLocked == false) {
-        lzt::set_power_limits_ext(
-            p_power_handle, &count_power,
-            power_limits_descriptors.data()); // set power limits for all
-                                              // descriptors
-
-        zes_power_limit_ext_desc_t power_sustained_get = {};
-
-        auto power_limits_descriptors_get =
-            lzt::get_power_limits_ext(p_power_handle, &count_power);
-        for (auto p_power_limits_descriptor_get :
-             power_limits_descriptors_get) {
-          if (p_power_limits_descriptor_get.level ==
-              ZES_POWER_LEVEL_SUSTAINED) {
-            power_sustained_get = p_power_limits_descriptor_get;
+      if (power_sustained_set.level == ZES_POWER_LEVEL_SUSTAINED) {
+        if (power_sustained_set.limitValueLocked == false) {
+          lzt::set_power_limits_ext(
+              p_power_handle, &count_power,
+              power_limits_descriptors.data()); // set power limits for all
+                                                // descriptors
+
+          zes_power_limit_ext_desc_t power_sustained_get = {};
+
+          auto power_limits_descriptors_get =
+              lzt::get_power_limits_ext(p_power_handle, &count_power);
+          for (auto p_power_limits_descriptor_get :
```

Comment:
minor: could we use const auto& in this for loop and all others. Other than that comment, PR looks fine.

Reply:
done